### ISSUE TITLE
OSError exception when starting mpsyt without mplayer or mpv installed

### ISSUE THREAD

**Issue by GijsTimmers** [np1/mps-youtube#263]
Running mps-youtube v0.2.4 on Ubuntu 15.04.

```
[me@pc ~]$ mpsyt
Traceback (most recent call last):
  File "/usr/local/bin/mpsyt", line 7, in <module>
    from mps_youtube import main
  File "/usr/local/lib/python2.7/dist-packages/mps_youtube/__init__.py", line 4, in <module>
    init()
  File "/usr/local/lib/python2.7/dist-packages/mps_youtube/main.py", line 907, in init
    [Config.PLAYER.get, "--list-options"]))
  File "/usr/lib/python2.7/subprocess.py", line 566, in check_output
    process = Popen(stdout=PIPE, *popenargs, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1335, in _execute_child
    raise child_exception
OSError: [Errno 2] Bestand of map bestaat niet
```

After installing `mpv`, this error disappeared.

**Comment by ids1024** [comment]
Should be fixed by commit 08c37986023bd7f12f2d789b8fb8e5de541177cb.

**Comment by GijsTimmers** [comment]
Thank you!
